src/server/rest/v1/validator/TenantValidation.ts

```diff
@@ -40,6 +46,24 @@ export default class TenantValidator extends SchemaValidator {
     return tenant;
   }
 
+  public validateTenantDeleteRequestSuperAdmin(request: any): string {
+    // Validate schema
+    this.validate(this._tenantDeleteReqSuperAdmin, request);
+    return request.ID;
+  }
+  
+  public validateGetLogoReqSuperAdmin(request: any): string {
+    // Validate schema
+    this.validate(this._tenantGetLogoReqSuperAdmin, request);
+    return request.ID;
+  }
+
+  public validateTenantGetReqSuperAdmin(request: any): string {
+    // Validate schema
+    this.validate(this._tenantGetReqSuperAdmin, request);
+    return request;
```

Comment:
You return the whole request here. Not the id, which you actually need where you call it. As a result you are not able to edit a tenant anymore.